Stream_support/include/CGAL/IO/io.h

```diff
@@ -176,8 +176,7 @@ class Input_rep<double> : public IO_rep_is_specialized {
 	break;
       }
     }while(true);
-
-    if(sscanf(buffer.c_str(), "%lf", &t) != 1) {
+    if(sscanf_s(buffer.c_str(), "%lf", &t) != 1) {
```

Comment:
That will probably not compile with a few compilers, because it is not part of the C++ standard, as far as I know, and only part of the C11 standard.

Reply:
It is protected by ` #ifdef _MSC_VER`